unittests/CMakeLists.txt

```diff
@@ -27,3 +27,6 @@ add_test( NAME test_tracing_node COMMAND python ${CMAKE_CURRENT_LIST_DIR}/test_t
 add_test( NAME test_reward_synapse COMMAND python ${CMAKE_CURRENT_LIST_DIR}/test_reward_synapse.py )
 add_test( NAME test_reward_synapse_io COMMAND python ${CMAKE_CURRENT_LIST_DIR}/test_reward_synapse_io.py )
 add_test( NAME test_reward_in_proxy COMMAND python ${CMAKE_CURRENT_LIST_DIR}/test_reward_in_proxy/test.py )
+
+# Integration Tests
+add_test( NAME test_pattern_matching COMMAND python ${CMAKE_CURRENT_LIST_DIR}/test_pattern_matching.py )
```

Comment:
I would not call it "test_pattern_matching". Maybe something that describes better what it does like "test_music_integration" or so...